TestProject.BL/Mappers/PostMapper.cs

```diff
@@ -6,15 +6,15 @@ namespace TestProject.BL.Mappers
 {
     public static class PostMapper
     {
-        public static Post MapPostEditorModelToPost(PostEditorModel postEditorModel, int userId)
+        public static Post MapPostEditorModelToPost(PostEditorModel postEditorModel, int userId, 
+            DateTime createDate, DateTime updateDate)
```

Comment:
Setting parameters those are not a part of initial model - is not a responsibility of a mapper. It should convert one model to the other and nothing more. There should not be any parameters except initial model.
Add missing properties to the BL model.